src/core-asyncpipeline/AsyncPipeline/AsyncPipeline.T/AsyncPipeline.T.cs

```diff
@@ -0,0 +1,25 @@
+#nullable enable
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    public readonly partial struct AsyncPipeline<T> : IEquatable<AsyncPipeline<T>>
+    {
+        private static CancellationToken CanceledToken() => new(canceled: true);
+
+        private readonly ValueTask<T> valueTask;
+
+        private readonly bool isCanceled;
+
+        private readonly CancellationToken cancellationToken;
+
+        internal AsyncPipeline(ValueTask<T> valueTask, bool isCanceled, CancellationToken cancellationToken)
+        {
+            this.valueTask = valueTask;
+            this.cancellationToken = cancellationToken;
+            this.isCanceled = isCanceled;
```

Comment:
Let's swap order of `isCanceled` and `cancellationToken` init to accord with the fields/params declaration order.

Reply:
Done